docs/en/observability/apm/monitor-apm-server/monitor.asciidoc

```diff
@@ -24,6 +24,15 @@ To learn how, see {ref}/configuring-metricbeat.html[Collect {es} monitoring data
 Alternatively, open the **{stack-monitor-app}** app in {kib} and follow the in-product guide.
 ****
 
+[WARNING]
+====
+*Only use this option if you have _not_ yet enrolled {agent}.*
+If you edit and re-enroll {agent}, you will likely lose the state of the agent.
+
+If you have already enrolled the {agent}, consider using the Fleet UI to change the agent's HTTP monitoring endpoint
```

Comment:
Ideally we would have:

- For versions pre 8.15 we have to say:
```
*This procedure works only if you've _not_ yet enrolled {agent}.*
If you've already enrolled {agent}, you'll need to re-enroll {agent} and you'll lose the state of the agent.
```

- For version 8.15 and more recent
```
Use the Fleet UI to change the agent's HTTP monitoring endpoint using {fleet-guide}/agent-policy.html#advanced-agent-monitoring-settings[Advanced agent monitoring settings].
```

Reply:
@simitt is someone from your team available to take a look at this recommendation?

Reply:
@raultorrecilla know better about the team's availability.